package.json

```diff
@@ -42,40 +42,28 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@changesets/cli": "^2.26.0",
+    "@changesets/cli": "^2.27.1",
     "chokidar-cli": "^3.0.0",
-    "concurrently": "^7.6.0",
-    "cross-env": "^7.0.3",
-    "dotenv-cli": "7.0.0",
-    "husky": "^8.0.2",
-    "lint-staged": "^13.2.1",
-    "patch-package": "~6.5.1",
+    "concurrently": "^8.2.2",
+    "husky": "^9.0.10",
+    "lint-staged": "^15.2.2",
+    "patch-package": "~8.0.0",
     "prettier": "~3.1.0",
-    "rimraf": "^3.0.2",
+    "rimraf": "^5.0.5",
     "serve": "^14.2.1",
-    "turbo": "~1.6.3"
+    "turbo": "~1.12.3"
   },
   "resolutions": {
```

Comment:
Just curious, how did you know/decide which resolutions were removable?

Reply:
I went through the lockfile and also ran `npm ls` for each dep. Removed the ones that were no longer necessary. Typically this means the dep is no longer being used, or the requested ranges are already satisfactory.